src/main/java/org/alfresco/repo/site/SiteMembersCannedQueryParams.java

```diff
@@ -1,55 +1,61 @@
-/*
- * #%L
- * Alfresco Repository
- * %%
- * Copyright (C) 2005 - 2016 Alfresco Software Limited
- * %%
- * This file is part of the Alfresco software. 
- * If the software was purchased under a paid Alfresco license, the terms of 
- * the paid license agreement will prevail.  Otherwise, the software is 
- * provided under the following open source license terms:
- * 
- * Alfresco is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Lesser General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- * 
- * Alfresco is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Lesser General Public License for more details.
- * 
- * You should have received a copy of the GNU Lesser General Public License
- * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
- * #L%
- */
+/*
+ * #%L
+ * Alfresco Repository
+ * %%
+ * Copyright (C) 2005 - 2016 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software. 
+ * If the software was purchased under a paid Alfresco license, the terms of 
+ * the paid license agreement will prevail.  Otherwise, the software is 
+ * provided under the following open source license terms:
+ * 
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
 package org.alfresco.repo.site;
 
 /**
  * Conveys parameters for the site members canned query.
- * 
+ *
  * @author steveglover
  *
  */
 public class SiteMembersCannedQueryParams
 {
 	private String shortName;
-	private boolean collapseGroups;
+	final private boolean expandGroups;
 
-	public SiteMembersCannedQueryParams(String shortName, boolean collapseGroups)
+	public SiteMembersCannedQueryParams(String shortName, boolean expandGroups)
 	{
 		super();
 		this.shortName = shortName;
-		this.collapseGroups = collapseGroups;
+		this.expandGroups = expandGroups;
 	}
 
 	public String getShortName()
 	{
 		return shortName;
 	}
 
+	/** deprecated from 7.0.0 use isExpandGroups method instead */
 	public boolean isCollapseGroups()

```

Comment:
Since this is not an AlfrescoPublicAPI, we can change it. We should remove this method immediately